app/assets/stylesheets/conversation_index.scss

```diff
@@ -0,0 +1,76 @@
+@mixin shadow1 {
+  box-shadow: 1px 1px 1px #888888;
+}
+@mixin shadow2 {
+  box-shadow: 2px 2px 2px #888888;
+}
+@mixin shadow3 {
+  box-shadow: 3px 3px 3px #888888;
+}
+@mixin shadow4 {
+  box-shadow: 4px 4px 4px #888888;
+}
+
+.conversation-container {
+  @include shadow2;
+  background-color: $grey-light1;
+  margin: 10px 10px 10px 10px;
```

Comment:
Properties should be ordered background-color, list-style, margin, padding<br>Shorthand form for property `margin` should be written more concisely as `10px` instead of `10px 10px 10px 10px`